firefox-ios/nimbus-features/bookmarkRefactorFeature.yaml

```diff
@@ -14,5 +14,5 @@ features:
           enabled: false
       - channel: developer
         value:
-          enabled: false
+          enabled: true
```

Comment:
Could we disable the feature flag and turn it on later when the tests are ready?

@isabelrios Could you please provide us some guidance on what to do on the bookmark refactor work? We probably need to update a number of tests.

Reply:
Conversataion on slack, let's sync up today to stablish a plan